cmd/oras/internal/option/spec.go

```diff
@@ -17,60 +17,69 @@ package option
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/spf13/pflag"
 	"oras.land/oras-go/v2"
 	oerrors "oras.land/oras/cmd/oras/internal/errors"
 )
 
 const (
-	ImageSpecV1_1    = "v1.1"
-	ImageSpecV1_0    = "v1.0"
-	ReferrersTagV1_1 = "v1.1-referrers-tag"
-	ReferrersAPIV1_1 = "v1.1-referrers-api"
+	ImageSpecV1_1                    = "v1.1"
+	ImageSpecV1_0                    = "v1.0"
+	DistributionSpecReferrersTagV1_1 = "v1.1-referrers-tag"
+	DistributionSpecReferrersAPIV1_1 = "v1.1-referrers-api"
 )
```

Comment:
Since they are different specs, would it be better to split them into two blocks?